src/main/java/com/datasift/client/pylon/DataSiftPylon.java

```diff
@@ -1,10 +1,6 @@
 package com.datasift.client.pylon;
 
-import com.datasift.client.BaseDataSiftResult;
-import com.datasift.client.DataSiftApiClient;
-import com.datasift.client.DataSiftConfig;
-import com.datasift.client.DataSiftResult;
-import com.datasift.client.FutureData;
+import com.datasift.client.*;
```

Comment:
Can you remove the wildcard import please. Check style is configured not to allow them